pkg/overlay/server.go

```diff
@@ -67,7 +71,7 @@ func (o *Server) FindStorageNodes(ctx context.Context, req *pb.FindStorageNodesR
 	result := []*pb.Node{}
 	for {
 		var nodes []*pb.Node
-		nodes, start, err = o.populate(ctx, start, maxNodes, restrictedBandwidth, restrictedSpace, excluded)
+		nodes, start, err = o.populate(ctx, req.Start, maxNodes, restrictedBandwidth, restrictedSpace, excluded)
```

Comment:
nit -- `req.GetStart()`

Reply:
What happens if Start isn't specified ?

Reply:
i think it's fine since it was unspecified originally